checkov/cloudformation/runner.py

```diff
@@ -234,7 +234,9 @@ def get_graph_checks_report(self, root_folder: str, runner_filter: RunnerFilter)
         return report
 
     def extract_images(
-        self, graph_connector: DiGraph | None = None, definitions: dict[str, dict[str, Any] | list[dict[str, Any]]] | None = None
+        self, graph_connector: DiGraph | None = None,
+            definitions: dict[str, dict[str, Any] | list[dict[str, Any]]] | None = None,
+            definitions_raw: dict[str, list[tuple[int, str]]] | None = None
```

Comment:
```suggestion
        self,
        graph_connector: DiGraph | None = None,
        definitions: dict[str, dict[str, Any] | list[dict[str, Any]]] | None = None,
        definitions_raw: dict[str, list[tuple[int, str]]] | None = None
```
just put them all on the same indent, looks somehow weird 😄